src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -1,34 +1,10 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
 <Project>
-  <UsingTask TaskName="Microsoft.DotNet.PackageValidation.ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
+  <PropertyGroup>
+    <DotNetPackageValidationAssembly Condition="'$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)..\tools\net472\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
```

Comment:
IT seems like there is a property that points to the directory where the tasks are already, it is defined in the Common targets and we are imported after them so it should work?

https://github.com/dotnet/sdk/blob/3c35324c260eb11143a1c647169f21b8e4632d45/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets#L20

Reply:
It doesnt get defined for multi-targeting.

Reply:
This file gets imported in both multi target and single target:

https://github.com/dotnet/sdk/blob/3c35324c260eb11143a1c647169f21b8e4632d45/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.CrossTargeting.targets#L14

https://github.com/dotnet/sdk/blob/3c35324c260eb11143a1c647169f21b8e4632d45/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L12